catalog-info.yaml

```diff
@@ -0,0 +1,30 @@
+# Backstage documentation
+# https://backstage.io/docs/features/software-catalog/descriptor-format/
+
+# !!! WARNING !!!
+# This is a template file with a number of fields that need to be filled before merging this to the default branch
+apiVersion: backstage.io/v1alpha1
+# Component, API, Template, Group, User, Resource, System, Domain, Location
+kind: unknown
```

Comment:
@roryscarson how would you categorize an OSS Design System repository in Backstage?

Reply:
That's really up to y'all. https://backstage.io/docs/features/software-catalog/descriptor-format/ has descriptions of what each type is used for, but we don't really check this for anything I'm aware of. System, Component, or Resource would likely all work just fine. I vaguely remember talking with mechagodzilla about this and the outcome being "it's a best effort fit". @contentful/team-mechagodzilla this might be something to look into creating some docs for.